CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
 project(ndt_omp)
 
-# should use march=native ?
-add_definitions(-std=c++14 -msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2)
-set(CMAKE_CXX_FLAGS "-std=c++14 -msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2")
+add_definitions(-std=c++14)
+set(CMAKE_CXX_FLAGS "-std=c++14")
```

Comment:
I think you don't have to write the same options twice.

```
add_definitions(-std=c++14)
set(CMAKE_CXX_FLAGS "-std=c++14")
```

Reply:
If this compile option is changed, will it affect SIMD?

Reply:
@yukkysaito Thank you! Is it the same comment as this?
https://github.com/tier4/ndt_omp/pull/8#discussion_r783663368

Or do you mean `std=c++14`? (I assume not.)

Reply:
@kenji-miyake 

>Is it the same comment as this?

Ops! it is same thing. :pray:

Reply:
> I think you don't have to write the same options twice.
> 
> ```
> add_definitions(-std=c++14)
> set(CMAKE_CXX_FLAGS "-std=c++14")
> ```

In the previous comment, you said it is good to create another PR to modify it.
https://github.com/tier4/ndt_omp/pull/8#discussion_r783672936

Reply:
I see, it was written twice in the original code as well. I'm sorry I overlooked it.